internal/i18n/locales/en/default.po

```diff
@@ -0,0 +1,16 @@
+msgid ""
+msgstr ""
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

Comment:
what is this Plural-Forms doing?

Reply:
Nothing right now. But if you ever had a sentence like:

> There were %d bananas

You'd want to define:

- `=0` "There were no bananas"
- `=1` "There was 1 banana"
- `>1` "There were %d bananas"

That's what plurals do. It's part of the gettext standard: https://www.gnu.org/software/gettext/manual/html_node/Translating-plural-forms.html